instrumentation/github.com/jackc/pgx/splunkpgx/sql.go

```diff
@@ -0,0 +1,120 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package splunkpgx provides instrumentation for the github.com/jackc/pgx
+// package when using database/sql.
+//
+// To use this package, replace any blank identified imports of the
+// github.com/jackc/pgx package with an import of this package and
+// use the splunksql.Open function as a replacement for any sql.Open function
+// use. For example, if your code looks like this to start.
+//
+//     import (
+//     	"database/sql"
+//     	_ "github.com/jackc/pgx"
```

Comment:
It should be

```suggestion
	_ "github.com/jackc/pgx/v4/stdlib"
```

or 

```suggestion
	_ "github.com/jackc/pgx/stdlib"
```

before Go Modules were introduced (should we even handle such old versions as https://github.com/jackc/pgx/releases/tag/v3.5.0 ?)

Reply:
This is not what the [README](https://github.com/jackc/pgx#example-usage) or [wiki](https://github.com/jackc/pgx/wiki/Getting-started-with-pgx#hello-world-from-postgresql) include. Do you have an example or docs showing this use of `stdlib`?

Reply:
The README and wiki shows non-`database/sql"` API. You can test it 😉

Reply:
https://github.com/jackc/pgx/blob/master/stdlib/sql.go